src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +24,22 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
+    public static final int MAX_LENGTH = 5;
+
+    public String getUniqueCharacters(String fileName) throws IOException {
+        Set<Character> treeSet = new TreeSet<>();
+        try (BufferedReader reader = new BufferedReader(new FileReader(fileName))) {
+            while (reader.ready()) {
+                treeSet.add((char) Character.toLowerCase(reader.read()));
```

Comment:
suggestion: use readLine instead of read. 

http://qaru.site/questions/144797/why-is-bufferedreader-read-much-slower-than-readline